docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -3144,12 +3144,14 @@ trim_array(array, n)
 
 ### `range`
 
-Returns an Arrow array between start and stop with step. `SELECT range(2, 10, 3) -> [2, 5, 8]`
+Returns an Arrow array between start and stop with step. `SELECT range(2, 10, 3) -> [2, 5, 8]` or `SELECT range(DATE '1992-09-01', DATE '1993-03-01', INTERVAL '1' MONTH);`
 
 The range start..end contains all values with start <= x < end. It is empty if start >= end.
 
 Step can not be 0 (then the range will be nonsense.).
 
+Note that when the required range is a number, it accepts 1, 2, and 3 parameters, but when the required range is a date, it must be 3 non-NULL parameters.
```

Comment:
That somewhat unclear, can we have some examples?

Reply:
Sure